scripts/check-health.sh

```
@@ -0,0 +1,214 @@
+echo "========================================================"
+
+# Navigate to the Docker directory
+cd "$DOCKER_DIR" || {
+    echo -e "${RED}Error: Failed to change to Docker directory: $DOCKER_DIR${NC}"
+    exit 1
+}
+
+# Function to check service health
+check_service() {
+    local service=$1
+    local status=$(docker compose -f docker-compose.yml ps --format json "$service" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)